lib/components/album_image.dart

```diff
@@ -219,3 +219,75 @@ class _AlbumImageErrorPlaceholder extends StatelessWidget {
     );
   }
 }
+
+class TapToZoomImage extends StatefulWidget {
+  final AlbumImage albumImg;
```

Comment:
let's try to avoid abbreviations. Make widget and argument names as long and descriptive as needed :)  

But I do like the wrapper approach, that may be a bit nicer than always using boolean flags ^^